tests/utils/invalids.py

```diff
@@ -55,9 +55,11 @@
     {"rhc_config_state": "x"*12},
     {"cpu_model": "x"*101},
     {"number_of_cpus": "35465"},
+    {"number_of_cpus": 4294967296},
```

Comment:
Could you also add invalids for the other fields you introduced limits on?
```suggestion
    {"number_of_cpus": 4294967296},
    {"number_of_sockets": 4294967296},
    {"cores_per_socket": 4294967296},
```